lib/hijack/error.js

```diff
@@ -58,12 +58,17 @@ export function TrackUnhandledRejections () {
   process.on('unhandledRejection', function (reason) {
     // skip errors with `_skipKadira` flag
     if (
+      reason &&
       reason._skipKadira ||
       !Kadira.options.enableErrorTracking
     ) {
       return;
     }
 
+    if (!reason) {
```

Comment:
This needs to make sure `reason` is undefined, since other falsey values are also valid.